honeybee_radiance_command/options/rpict.py

```diff
@@ -0,0 +1,548 @@
+        self._w = BoolOption("w", "warning messages - default: on")
+        self._i = BoolOption("i", "irradiance calculation - default: off")
+        self._u = BoolOption(
+            "u", "correlated quasi-Monte Carlo sampling - default: off"
+        )
+        self._bv = BoolOption("bv", "back face visibility - default: on")
+        self._dt = NumericOption("dt", "direct threshold - default: 0.050000")
+        self._dc = NumericOption("dc", "direct certainty - default: 0.500000")
+        self._dj = NumericOption("dj", "direct jitter - default: 0.000000")
+        self._ds = NumericOption("ds", "direct sampling - default: 0.250000")
+        self._dr = IntegerOption("dr", "direct relays - default: 1")
+        self._dp = IntegerOption("dp", "direct pretest density - default: 512")
+        self._dv = BoolOption("dv", "direct visibility - default: on")
+        self._ss = NumericOption("ss", "specular sampling - default: 1.000000")
+        self._st = NumericOption("st", "specular threshold - default: 0.150000")
+        self._av = TupleOption(
+            "av", "ambient value - default: 0.000000 0.000000 0.000000", 3, float
+        )
+        self._aw = IntegerOption("aw", "ambient value weight - default: 0")
+        self._ab = IntegerOption("ab", "ambient bounces - default: 0")
+        self._aa = NumericOption("aa", "ambient accuracy - default: 0.200000")
+        self._ar = IntegerOption("ar", "ambient resolution - default: 64")
+        self._ad = IntegerOption("ad", "ambient divisions - default: 512")
+        self._as_ = IntegerOption("as_", "ambient super-samples - default: 128")
+        self._me = TupleOption(
+            "me",
+            "mist extinction coefficient - default: 0.00e+00 0.00e+00 0.00e+00",
+            3,
+            float,
+        )
+        self._ma = TupleOption(
+            "ma",
+            "mist scattering albedo - default: 0.000000 0.000000 0.000000",
+            3,
+            float,
+        )
+        self._mg = NumericOption(
+            "mg", "mist scattering eccentricity - default: 0.000000"
+        )
+        self._ms = NumericOption("ms", "mist sampling distance - default: 0.000000")
+        self._lr = IntegerOption("lr", "limit reflection - default: 7")
+        self._lw = NumericOption("lw", "limit weight - default: 1.00e-03")
+        self._am = NumericOption("am", "max photon search radius - default: 0.0")
+        self._on_setattr_check = True
+
+    @property
+    def vt(self):
+        """view type perspective - default: vtv"""
+        return self._vt
+
+    @vt.setter
+    def vt(self, value):
+        self._vt.value = value
+
+    @property
+    def vp(self):
+        """view point - default: 0.000000 0.000000 0.000000"""
+        return self._vp
+
+    @vp.setter
+    def vp(self, value):
+        self._vp.value = value
+
+    @property
+    def vd(self):
+        """view direction - default: 0.000000 1.000000 0.000000"""
```

Comment:
We need to update all the docstrings similar to how it is done in rtrace.

https://github.com/ladybug-tools/honeybee-radiance-command/blob/c9b1c59fb1c17dcc84f3e1a9fc144acc8309fa43/honeybee_radiance_command/options/rtrace.py#L216-L229

That will also help you to understand the options better and start writing the `_on_setattr` method to ensure user doesn't create invalid Radiance options for rpict.

https://github.com/ladybug-tools/honeybee-radiance-command/blob/c9b1c59fb1c17dcc84f3e1a9fc144acc8309fa43/honeybee_radiance_command/options/rtrace.py#L98-L131

Reply:
Where are we setting the default values for these options in rtrace?

Reply:
We are not! Radiance does.